Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -3452,4 +3452,23 @@ Document
             ])
         }
     }
+    
+    func testThematicBreak() throws {
+        let source = """
+
+        ---
+
+        """
+        
+        let markup = Document(parsing: source, options: .parseBlockDirectives)
+        let (bundle, context) = try testBundleAndContext(named: "TestBundle")
+        var contentTranslator = RenderContentCompiler(context: context, bundle: bundle, identifier: ResolvedTopicReference(bundleIdentifier: bundle.identifier, path: "/TestTutorial", sourceLanguage: .swift))
```

Comment:
minor: This path doesn't seem related to the content.

Reply:
Renamed! 😅